content/en/platform/corda/4.10/enterprise/cordapps/api-confidential-identity.md

```diff
@@ -46,7 +46,6 @@ and the counterparty to their new confidential identities. In the future, this f
 identities with multiple parties at once.
 
 You can see an example of using `SwapIdentitiesFlow` in <a href="../../../../../../en/api-ref/corda/4.10/community/kotlin/corda/net.corda.finance.flows/-two-party-deal-flow/index.md">`TwoPartyDealFlow.kt`</a> .
```

Comment:
Should this be a relref as well?

Reply:
Good spot. Unfortunately this is a different type and it also seems to be a unique type too. It is a relative link written in html (href) format, but with an .md ending. It looks like one of a kind and I did fiddle with it a bit on Friday to fix it, but to no avail. I will circle back to this one, along with the other random link types at the end, but I don't want to get stuck in the mire on outliers. Thanks Paul